docs/built_rst/csv/elements/polling_location.rst

```diff
@@ -7,61 +7,73 @@ polling_location
+|                                      |                          |              |              | this polling location.                   | present, then the implementation is      |
+|                                      |                          |              |              |                                          | required to ignore it.                   |
++--------------------------------------+--------------------------+--------------+--------------+------------------------------------------+------------------------------------------+
+| name                                 | ``xs:string``            | Optional     | Single       | Name of the polling location.            | If the field is invalid or not present,  |
+|                                      |                          |              |              |                                          | then the implementation is required to   |
+|                                      |                          |              |              |                                          | ignore it.                               |
++--------------------------------------+--------------------------+--------------+--------------+------------------------------------------+------------------------------------------+
+| photo_uri                            | ``xs:string``            | Optional     | Single       | Contains a link to an image of the       | If the field is invalid or not present,  |
+|                                      |                          |              |              | polling location.                        | then the implementation is required to   |
+|                                      |                          |              |              |                                          | ignore it.                               |
++--------------------------------------+--------------------------+--------------+--------------+------------------------------------------+------------------------------------------+
 
 .. code-block:: csv-table
    :linenos:
 
 
-    id,name,address_line,directions,hours,photo_uri,hours_open_id,is_drop_box,is_early_voting,latitude,longitude,latlng_source
-    poll001,ALBERMARLE HIGH SCHOOL","2775 Hydraulic Rd Charlottesville, VA 22901,Use back door,7am-8pm,www.picture.com,ho001,false,true,38.0754627,78.5014875,Google Maps
+    id,name,address_line,structured_location_name,structured_line_1,structured_city,structured_state,structured_zip,directions,hours,photo_uri,hours_open_id,is_drop_box,is_early_voting,latitude,longitude,latlng_source
+    poll001,ALBERMARLE HIGH SCHOOL","2775 Hydraulic Rd Charlottesville, VA 22901,ALBERMARLE HIGH SCHOOL,2775 Hydraulic Rd,Charlottesville,VA,22901,Use back door,7am-8pm,www.picture.com,ho001,false,true,38.0754627,78.5014875,Google Maps
```

Comment:
This doesn't seem to be a valid example. It has both address_line and the structured address. Plus I'm not sure what the quotations are doing here.

Reply:
Will clean this up. It is a bad example.